content/en/docs/architecture/release-gating.md

```diff
@@ -67,88 +36,215 @@ Good candidates include:
 
-Not all test scenarios and test suites are considered release gating. On a given day we may generate tens of release
-payloads, but not choose to execute or iterate the full set of suites on those jobs. Also, some scenarios (such as
-upgrade rollback) are intended to generate statistical signal (a given rollback may succeed or fail due to factors
-outside the test control). All release informing jobs should start as periodics (as described below) and some may be
-promoted to be “release-triggered” jobs.
 
-### Process for Extending the Release Gating Suite
 
-Our goal of weekly z-stream releases necessitates that release-gating tests provide stable and unambiguous signal about
-regressions in the product or the projects and projects on top. To provide this level of stability and observability,
-any new release-blocking tests must follow the below steps:
 
-#### Add A Periodic Job
+### Process for creating a new job
 
-Create [a new multi-stage job](/docs/architecture/step-registry/) on top of latest OCP that runs as a periodic job. The
-job should reuse as much of the existing step registry as possible, and must include either the standard gather and
-gather-extra steps from the step registry, or custom steps (in rare cases) that gather as much of the same data as possible.
+Create [a new multi-stage job](/docs/architecture/step-registry/) on top of latest OCP that
+runs as a periodic job. 
 
-This does not block the release but you can run it often enough to get a decent signal. At least once a day, but every 6
-or 12 hours is better.
+Periodic jobs must:
 
-The job name should follow the [naming guidelines](/docs/how-tos/naming-your-ci-jobs/).
+- follow the [naming guidelines](/docs/how-tos/naming-your-ci-jobs/)
+- be attached to release branch (release-X.Y) - do not add periodics to your main or 
+  master branch unless unavoidable
+- kept separate from any presubmits by using the variants feature of ci-operator
+- use the release payload provided by CI in the `RELEASE_IMAGE_LATEST` and
+`RELEASE_IMAGE_INITIAL` environment variables; do not hard code any particular release
+image
+- re-use as much of the existing step registry as possible
+- include standard `gather` and `gather-extra` steps, or in rare cases custom steps 
+  that gather as much of the same data as possible
+
+You should initially run the job on a frequent basis to get a good sample to determine 
+it's reliability; at least daily. Frequency can be adjusted later depending 
+on the path for release blocking you choose. Job pass rates can be monitored in [Sippy](https://sippy.dptools.openshift.org/sippy-ng/).
 
 Where possible you should be running this same test on PRs to your project’s repos to make sure changes to your project
 don’t result in downstream breakages to the release job. We recognize this may not be an option for all projects.
 
-#### Becoming Informing
+### Becoming Blocking in Component Readiness
```

Comment:
I'd like either myself or nick/nick's delegate to approve being in the standard component readiness view.  For the foreseeable future, it's more of a business/org decision than an algorithmic one.